docs/index.rst

```diff
@@ -36,6 +36,7 @@ Features
    userGuide
    useAzkaban
    eventTrigger
+   ajaxApi
```

Comment:
Is this the right order? I'm not sure about where should we put the event trigger doc.